src/index.js

```diff
@@ -0,0 +1,185 @@
+        sky = skies.sunny;
+        skyColor = 'sunny';
+    } else if (inputSky === 'Cloudy') {
+        sky = skies.cloudy;
+        skyColor = 'cloudy';
+    } else if (inputSky === 'Rainy') {
+        sky = skies.rainy;
+        skyColor = 'rainy';
+    } else if (inputSky === 'Snowy') {
+        sky = skies.snowy;
+        skyColor = 'snowy';
+    }
+
+    skyBox.textContent = sky;
+    const gardenContent = document.getElementById('gardenContent');
+    gardenContent.classList = `garden__content ${skyColor}`;
+}
+
+const formatTempAndGarden = () => {
+    const landscapes = {
+        hot: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+        warm: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+        cool: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+        cold: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    };
+    let temp = state.temp;
+    let color = 'green';
+    let landscape = landscapes.warm;
+
+    if (temp >= 100) {
+        color = 'red';
+        landscape = landscapes.hot;
+    } else if (temp >= 95) {
+        color = 'orangered';
+        landscape = landscapes.hot;
+    } else if (temp >= 86) {
+        color = 'orange';
+        landscape = landscapes.hot;
+    } else if (temp >= 77) {
+        color = 'green';
+        landscape = landscapes.warm;
+    } else if (temp >= 68) {
+        color = 'darkgreen';
+        landscape = landscapes.warm;
+    } else if (temp >= 59) {
+        color = 'teal';
+        landscape = landscapes.cool;
+    } else if (temp >= 50) {
+        color = 'cornflowerblue';
+        landscape = landscapes.cool;
+    } else if (temp >= 41) {
+        color = 'steelblue';
+        landscape = landscapes.cold;
+    } else if (temp >= 32) {
+        color = 'royalblue';
+        landscape = landscapes.cold;
+    } else if (temp >= 23) {
+        color = 'darkblue';
+        landscape = landscapes.cold;
+    } else if (temp >= 14) {
+        color = 'darkslateblue';
+        landscape = landscapes.cold;
+    } else if (temp >= 5) {
+        color = '#424040';
+        landscape = landscapes.cold;
+    } else {
+        color = 'black';
+        landscape = landscapes.cold;
+    }
+
+    const temperature = document.getElementById('tempValue');
+    const newLandscape = document.getElementById('landscape');
+    temperature.style.color = color;
+    temperature.textContent = String(state.temp);
+    newLandscape.textContent = landscape;
+};
+
+const increaseTemp = () => {
+    state.temp++;
+    formatTempAndGarden();
+};
+
+const decreaseTemp = () => {
+    state.temp--;
+    formatTempAndGarden();
+};
+
+const registerEventHandlers = () => {
+    formatTempAndGarden();
+    const currentTempBtn = document.getElementById('currentTempBtn');
+    currentTempBtn.addEventListener('click', getCoordinates);
+
+    const increaseTempBtn = document.getElementById('increaseTempBtn');
+    increaseTempBtn.addEventListener('click', increaseTemp);
+
+    const decreaseTempBtn = document.getElementById('decreaseTempBtn');
+    decreaseTempBtn.addEventListener('click', decreaseTemp);
+
+    updateCityName();
```

Comment:
This function call is not necessary when the page first loads since it's the one that needs to be called when users are typing something in to the search bar.